lib/api_endpoint/swagger2/api_swagger_final_path_processing.py

```diff
@@ -35,6 +35,12 @@ def process_output(
                 'basic_auth': {
                     'type': 'basic'}},
             'basePath': '/api',
+            'produces': [
+                'application/json'
+            ],
+            "consumes": [
+                "application/json"
+            ],
```

Comment:
I suggest we delegate this conversation to a seperate jira ticket (3802), because it is a bit vague for me - I see the old code filtered GET and DELETE and added consumes to the other http verbs. In the old specifications (prior refactoring) we have a global consumes + consumes for every verb except GET and DELETE (we don't have an empty consumes overwriting the global behavior) - I am not sure what are we striving to do - if we want to duplicate the old specifications I will just return the check and keep adding consumes to every non GET or DELETE verb, but this does not seem correct.

Reply:
Deferring this for another change is fine with me.

Old was broken, because GET and DELETE still got the consumes from the global definition. It can be overriden at individual operation level by providing empty consumes. This is explicitly mentioned in the OAS 2.0 specification. We must generate OpenAPI definitions which are compliant with the spec, I mean we have to make it right, not consistent with what it used to be (when that was broken).